airbyte-integrations/connectors/source-pokeapi-manifest/manifest.yaml

```diff
@@ -13,22 +13,34 @@ definitions:
       type: DeclarativeStream
       name: pokemon
       primary_key:
-        - id
+        - name
       retriever:
         type: SimpleRetriever
         requester:
           $ref: "#/definitions/base_requester"
-          path: /{{config['pokemon_name']}}
+          path: /
           http_method: GET
         record_selector:
           type: RecordSelector
           extractor:
             type: DpathExtractor
-            field_path: []
+            field_path:
+              - results
       schema_loader:
         type: InlineSchemaLoader
         schema:
           $ref: "#/schemas/pokemon"
+      transformations:
+        - class_name: "source_declarative_manifest.components.ListAddFields"
+          fields:
+            - path: ["bens_duplicated_field"]
+              value: "{{ record['name'] }}"
+              value_type: "string"
+        - class_name: "source_declarative_manifest.components.ListAddFields"
```

Comment:
The one interesting piece I was wondering about is how can we resolve just `components.%NAME%` into the file, omitting `source_declarative_manifest`. It's probably a small patch to sys paths?

Reply:
Oh im sure we can do that easily! Im not sure why we havent tbh!

Reply:
We can certainly access to module name.

There is some prior art
- https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/schema/json_file_schema_loader.py#L20-L24
- https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/utils/schema_helpers.py#L132

Happy to chat @ChristoGrab !